home.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="dist/css/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css"
+    />
+    <title>Pattern | Home</title>
+  </head>
+  <body>
+    <nav class="navbar bg-primary mb-1">
+      <div class="container-fluid" style="max-width: 800px">
+        <h1>
+          <a class="navbar-brand" href="home.html">pattern</a>
+        </h1>
+        <div class="navbar-nav d-flex flex-row align-items-center">
+          <a class="nav-link ms-4" aria-current="page" href="home.html">
+            <i class="bi bi-house-fill"></i>
+          </a>
+          <a class="nav-link ms-4" href="profile.html">
+            <i class="bi bi-person"></i>
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main class="mx-auto mb-5" style="max-width: 600px">
```

Comment:
```suggestion
    <main class="mx-auto mb-5">
```

You should not be using any inline styles. If you need to make a custom class, that's ok - but I think you can do this with row/col